### PR TITLE
Bug-Fix/FE-256-minor-UI-fixes-for-the-graph-viewer

### PR DESCRIPTION
**Pull request by ajurdak**
### Scope & Purpose

- Better spacing of attributes in edit dialog (esp when lots of attributes): After talking to @Simran-B we decided to remove editable attributes from the list and add the "tree"-mode to the editor.
- Re-order entries in right click menu
- Make Right click menu & toolbar work in full screen (for this, make the whole page fullscreen instead of a specific div)

- [x] :hankey: Bugfix

### Checklist

- [x] Manually tested
- [x] :book: CHANGELOG entry made (ToDo)
- [x] Backports
  - [x] Backport for 3.11: #18899

#### Related Information

- [x] GitHub issue / Jira ticket: https://arangodb.atlassian.net/browse/FE-256

### CODE REVIEW

File: js/apps/system/_admin/aardvark/APP/react/src/views/graphV2/graphModals/DeleteNodeModal.tsx [hunk 1]
```diff
@@ -62,7 +61,7 @@ const useDeleteNodeAction = ({ deleteEdges }: { deleteEdges: boolean }) => {
 export const DeleteNodeModal = () => {
   const { onClearAction } = useGraph();
   const [deleteEdges, setDeleteEdges] = useState(true);
-  const { nodeId, nodeData, deleteNode } = useDeleteNodeAction({
+  const { nodeId, deleteNode } = useDeleteNodeAction({
```
Comment (palashkaria) [inline js/apps/system/_admin/aardvark/APP/react/src/views/graphV2/graphModals/DeleteNodeModal.tsx:64]: why were attributes removed for both delete modals?

File: js/apps/system/_admin/aardvark/APP/react/src/views/graphV2/graphModals/DeleteEdgeModal.tsx [hunk 2]
```diff
@@ -40,7 +39,7 @@ const useDeleteEdgeAction = (selectedAction?: SelectedActionType) => {
 };
 export const DeleteEdgeModal = () => {
   const { selectedAction, onClearAction } = useGraph();
-  const { edgeId, edgeData, deleteEdge } = useDeleteEdgeAction(selectedAction);
+  const { edgeId, deleteEdge } = useDeleteEdgeAction(selectedAction);
```
Comment (palashkaria) [inline js/apps/system/_admin/aardvark/APP/react/src/views/graphV2/graphModals/DeleteEdgeModal.tsx:42]: why was this removed?

**Review by palashkaria — COMMENTED** [review]
Couple of queries

**Review by palashkaria — APPROVED** [review]

File: js/apps/system/_admin/aardvark/APP/react/src/views/graphV2/graphModals/DeleteEdgeModal.tsx (hunk 2 above)
Comment (palashkaria) [reply js/apps/system/_admin/aardvark/APP/react/src/views/graphV2/graphModals/DeleteEdgeModal.tsx:42]: this was removed because old viewer also doesn't have this, and it can look very messy with a lot of attributes

**Comment by KVS85** [comment]
- [ ] https://jenkins01.arangodb.biz/job/arangodb-ANY-jslint/29036/
